src/ui/RecipeTitleCard.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+import Flex from './Flex';
+
+const StyledFooter = styled.footer`
```

Comment:
the footer should live on the Page component, and it should be the same on all pages

Reply:
if you're just adding a buffer of space, i would prefer to set a height to pixels rather than gap. gap i associate with spacing between elements, even if it technically works here

Reply:
This is actually the gap between "by" and "<AuthorName>", lemme know if I should get rid of it anyway